### PR TITLE
Some mana nits

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: sdk/src/types/block/rand/mana.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::block::{mana::ManaAllotment, rand::output::rand_account_id};
+
+/// Generates a random mana allotment.
+pub fn rand_mana_allotment(mana: u64) -> ManaAllotment {
+    ManaAllotment::new(rand_account_id(), mana).unwrap()
```
Comment (Thoralf-M) [inline sdk/src/types/block/rand/mana.rs]: Why don't make the amount also random?

Comment (thibault-martinez) [reply sdk/src/types/block/rand/mana.rs]: I'm assuming that in future tests we'd need to provide an actual value to test edge cases

Comment (Thoralf-M) [reply sdk/src/types/block/rand/mana.rs]: But then just not use the random function? account id should probably also be tested in some cases

Comment (thibault-martinez) [reply sdk/src/types/block/rand/mana.rs]: Yeah ok, I'll change it later

Comment (thibault-martinez) [reply sdk/src/types/block/rand/mana.rs]: Changed

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/src/types/block/rand/mana.rs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::block::{
+    mana::ManaAllotment,
+    rand::{number::rand_number, output::rand_account_id},
+};
+
+/// Generates a random mana allotment.
+pub fn rand_mana_allotment() -> ManaAllotment {
+    ManaAllotment::new(rand_account_id(), rand_number()).unwrap()
```
Comment (Thoralf-M) [inline sdk/src/types/block/rand/mana.rs]: rand_number() should consider the theoretical mana max

Comment (thibault-martinez) [reply sdk/src/types/block/rand/mana.rs]: Done

**Review by kwek20 — COMMENTED** [review]
Theyre just called Allotment in the TIPs, did you request the rename there as well?

File: sdk/src/types/block/mana/allotment.rs [hunk 3]
```diff
@@ -8,34 +8,34 @@ use packable::{
     Packable,
 };
 
-use super::MAX_THEORETICAL_MANA;
+use super::THEORETICAL_MANA_MAX;
```
Comment (DaughterOfMars) [inline sdk/src/types/block/mana/allotment.rs:11]: I prefer the old name

Comment (thibault-martinez) [reply sdk/src/types/block/mana/allotment.rs:11]: I don't really care if it's a prefix or suffix but I do want consistency and most of our constants have a suffix, there is even a `COUNT_MAX` in the same file. There is a mix of both in the repository, if we all agree on what we prefer I can do a PR to change them all.

Comment (DaughterOfMars) [reply sdk/src/types/block/mana/allotment.rs:11]: bleh, okay, it's not that important

**Review by DaughterOfMars — APPROVED** [review]

**Review by abdulmth — APPROVED** [review]

File: sdk/src/types/block/mana/allotment.rs (hunk 3 above)
Comment (thibault-martinez) [reply sdk/src/types/block/mana/allotment.rs:11]: I opened an issue anyway, it's all a bit inconsistent atm

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Review by qrayven — APPROVED** [review]
